<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_divice_form</class>
 <widget class="QDialog" name="add_divice_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dodaj urządzenie</string>
  </property>
  <widget class="QPushButton" name="addDiviceButton">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>360</y>
     <width>91</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>457</x>
     <y>360</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Anuluj</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="divNameEdit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>131</width>
     <height>28</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="stopBitLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Bity stopu:</string>
   </property>
  </widget>
  <widget class="QLabel" name="parityLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Kontrola parzystości:</string>
   </property>
  </widget>
  <widget class="QLabel" name="speedTransLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Prędkość transmisji:</string>
   </property>
  </widget>
  <widget class="QLabel" name="divNameLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Nazwa urządzenia:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="parityCombo">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>140</y>
     <width>131</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="bitLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Bity danych:</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="commandTableWidget">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>40</y>
     <width>256</width>
     <height>161</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>false</bold>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="addCommandButton">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>290</y>
     <width>141</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Dodaj komendę</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="commandDescEdit">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>220</y>
     <width>171</width>
     <height>28</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="commandDescLabel">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>220</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Opis:</string>
   </property>
  </widget>
  <widget class="QLabel" name="commandLabel">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>250</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Komenda: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="commandEdit">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>250</y>
     <width>171</width>
     <height>28</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="stopBitLabel_4">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>20</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Zapisane komendy urządzenia</string>
   </property>
  </widget>
  <widget class="QComboBox" name="baudRateCombo">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>60</y>
     <width>131</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="dataBitsCombo">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>131</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="stopBitsCombo">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>180</y>
     <width>131</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="removeCommandButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>290</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Usuń</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
